packages/auth/src/graphql/resolvers/page.test.ts

```diff
@@ -0,0 +1,251 @@
+                  ${queryFields}
+                }
+              }
+            `,
+          variables: {
+            id: parent?.getId(),
+            input
+          }
+        })
+        .then(toConnection)
+      expect(query.edges).toHaveLength(10)
+      expect(query.pageInfo.hasNextPage).toBeTruthy()
+      expect(query.pageInfo.hasPreviousPage).toBeFalsy()
+      expect(query.pageInfo.startCursor).toEqual(models[0].id)
+      expect(query.pageInfo.endCursor).toEqual(models[9].id)
+    })
+
+    test('pageInfo is correct on pagination from middle', async (): Promise<void> => {
+      const models = await createModels()
+      const query = await apolloClient
+        .query({
+          query: parent
+            ? gql`
+                  query Page($id: String!, $after: String!) {
+                    ${parent.query}(id: $id) {
+                      ${pagedQuery}(input: {after: $after}) {
+                        ${queryFields}
+                      }
+                    }
+                  }`
+            : gql`
+                  query Page($after: String!) {
+                    ${pagedQuery}(input: {after: $after}) {
+                      ${queryFields}
+                    }
+                  }
+                `,
+          variables: {
+            id: parent?.getId(),
+            after: models[19].id
+          }
+        })
+        .then(toConnection)
+      expect(query.edges).toHaveLength(20)
+      expect(query.pageInfo.hasNextPage).toBeTruthy()
+      expect(query.pageInfo.hasPreviousPage).toBeTruthy()
+      expect(query.pageInfo.startCursor).toEqual(models[20].id)
+      expect(query.pageInfo.endCursor).toEqual(models[39].id)
+    })
+
+    test('pageInfo is correct on pagination near end', async (): Promise<void> => {
+      const models = await createModels()
+      const query = await apolloClient
+        .query({
+          query: parent
+            ? gql`
+                  query Page($id: String!, $after: String!) {
+                    ${parent.query}(id: $id) {
+                      ${pagedQuery}(input: {after: $after, first: 10}) {
+                        ${queryFields}
+                      }
+                    }
+                  }`
+            : gql`
+                  query Page($after: String!) {
+                    ${pagedQuery}(input: {after: $after, first: 10}) {
+                      ${queryFields}
+                    }
+                  }
+                `,
+          variables: {
+            id: parent?.getId(),
+            after: models[44].id
+          }
+        })
+        .then(toConnection)
+      expect(query.edges).toHaveLength(5)
+      expect(query.pageInfo.hasNextPage).toBeFalsy()
+      expect(query.pageInfo.hasPreviousPage).toBeTruthy()
+      expect(query.pageInfo.startCursor).toEqual(models[45].id)
+      expect(query.pageInfo.endCursor).toEqual(models[49].id)
+    })
+  })
+}
+
+test.todo('test suite must contain at least one test')
```

Comment:
```suggestion
```

Reply:
Hmm I think this broke the build 🤔

Reply:
I think you just need to `pnpm format`

Reply:
@mkurapov  If I remove that line, it creates a new error in test.
I had to add that again to remove errors in tests.